src/assemblyinfo/assemblyinfo.FSharp.Compiler.Private.dll.fs

```diff
@@ -71,4 +71,10 @@ open System.Runtime.InteropServices
 [<assembly: System.Reflection.AssemblyFileVersion("2017.06.27.0")>]
 #endif
 
+#if HAVE_VS_SDK
```

Comment:
Ugh, I was worried this might be needed

I'm not sure if I'm totally comfortable with this F# SDK DLL having a VS SDK attribute, but I guess it would only be there for the VS binary release of the DLL.  Perhaps it's necessary,

@brettfo Are there any downsides to having this attribute on a DLL besides the extra static dependencies?